changelog/7061.improvement.rst

```diff
@@ -0,0 +1 @@
+Raise a ValueError when a yielding fixture fails to yield a single value
```

Comment:
I think this is more of a bugfix than an improvement, so would change the changelog file to `7061.bugfix.rst`.

Reply:
apologies, i took the meta data from the issue which was not marked a bug, thanks will update

Reply:
No apologies required :)